### DISCUSSION TITLE
韓團BTS進軍NFT

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#259] [General]
https://udn.com/news/story/6811/5870237

最近BTS最大的新聞除了團員確診，就是他們進軍NTF市場了！最近一個月越來越多明星推出自己的NFT及購買並曬在社群網站上，不知道大家怎麼想的呢

**Comment by 107602505** [comment]
感覺這些明星進軍NFT市場又可以再賺更多的錢了

**Comment by JAIMEHUNG** [comment]
感覺擴大了藝能產業的產品市場，但小卡有動動和語音滿吸引人的(BTW: BTS是7位成員啦好好笑)

**Comment by cody910120** [comment]
怎麼大家都在搞NFT阿，真希望我也可以參一咖，可惜我沒錢哭哭

**Comment by Joan0518** [comment]
現在演藝圈很多不管大咖小咖的都出了NFT，感覺未來要搭配更多特色販售才會吸引人

**Comment by yt7l0** [comment]
有了NFT之後，商品越來越多元了XD

**Comment by vvvioletta** [comment]
NFT讓藝人的影響力和社群力更加強大了
